### PR TITLE
moved definition of statuses from front to backend

### PR DESCRIPTION
**Pull request by mertbagt**
receiving STATUSES from the backend instead of the front
single source of truth

### CODE REVIEW

**Comment by mertbagt** [comment]
put export on alphaSort and numSort in SortHook.js to use their functionality in additional places -> ScanOrder

**Review by ethanstrominger — APPROVED** [review]

File: src/components/ScanOrder.js [hunk 1]
```diff
@@ -77,50 +82,35 @@ const ScanOrder = (props) => {
...
   let nextDesc = "";
   let nextId = null;
   let nextSeq = null;
   let nextStatusFedOfficeCode = "";
+  let sortedStatuses = [];
 
-  if (STATUSES && order) {
-    STATUSES.sort(dynamicSort("sequence_num"));
+  if (statuses && order) {
+    sortedStatuses = numSort(statuses, "sequence_num", "asc");
```
Comment (rkaufman13) [inline src/components/ScanOrder.js:97]: It's probably good to be extra sure, but I'm surprised that the statuses would/could come out of the database unsorted..?

File: src/components/ScanOrder.js [hunk 2]
```diff
@@ -77,50 +82,35 @@ const ScanOrder = (props) => {
...
       }
-      if (i === STATUSES.length - 2) {
-        nextDesc = "FINAL";
-      }
     }
   }
 
   const handleUpdate = () => {
-    setOrder({
+    const upOrder = {
```
Comment (rkaufman13) [inline src/components/ScanOrder.js:113]: Extremely nitpicky but I'd "waste" a few extra characters to call this "updatedOrder" for readability

File: src/components/ScanOrder.js [hunk 3]
```diff
@@ -196,7 +196,7 @@ const ScanOrder = (props) => {
               <div className="form-group">
                 <label htmlFor="next_status">
                   Next Status:{" "}
-                  {STATUSES && order ? (
+                  {statuses && order.status.description ? (
                     <strong>
```
Comment (rkaufman13) [inline src/components/ScanOrder.js:200]: What about for Cancelled? How does that display here?

**Review by rkaufman13 — COMMENTED** [review]
Looks good to me w/a few teensy nitpicks!

File: src/components/ScanOrder.js (hunk 1 above)
Comment (mertbagt) [reply src/components/ScanOrder.js:97]: they come out of the database sorted by id, which is like the row number on a table.  I'd assume a future coder would write future statuses in a way that kept the sequence_id in order but I don't have a way to enforce that.

This way I'm not dependant on them being written meticulously.

File: src/components/ScanOrder.js (hunk 2 above)
Comment (mertbagt) [reply src/components/ScanOrder.js:113]: Makes sense, I'll do this in my next update as I don't want to risk having to reresolve conflicts if possible, but I will make a note to switch this as recommended

File: src/components/ScanOrder.js (hunk 3 above)
Comment (mertbagt) [reply src/components/ScanOrder.js:200]: I don't want people to accidentally roll into it.  Was planning on giving it a separate button all it's own but am open to feedback.

Comment (rkaufman13) [reply src/components/ScanOrder.js:200]: Ah I think I misunderstood what the code is doing here. No worries!

**Pull request merged**